rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1279,7 +1279,8 @@ stream {
             {{ $proxySetHeader }} X-Scheme               $pass_access_scheme;
 
             # Pass the original X-Forwarded-For
-            {{ $proxySetHeader }} X-Original-Forwarded-For {{ buildForwardedFor $all.Cfg.ForwardedForHeader }};
+            {{ $proxySetHeader }} X-Original-Forwarded-For  {{ buildForwardedFor $all.Cfg.ForwardedForHeader }};
+            {{ $proxySetHeader }} X-Original-Forwarded-Host {{ buildForwardedFor $all.Cfg.ForwardedHostHeader }};
```

Comment:
Hm, this is a bit confusing as you're using the `buildForwardedFor` template here. I know, it's correct as it's translating the configured header name, but it would be cool if you could come up with a clearer solution here. Also please add a comment about what you're doing here. 🙂

Reply:
I'll add the comment, sure!
what would you suggest?
1. renaming the `buildForwardedFor` template to something generic ? sounds a little risky as it's probably used with that name in custom templates.
2. I could create a new `buildForwardedHost` template function
3. I could implement a `replace` go template, so we could build the header as a combination of `toLower`, `replace`, and prepending the `http_` prefix.

Reply:
I'd go for option 2 (`buildForwardedHost`) here. At least it's safest, backward-compatible, non-breaking and we could benefit from having it in the future.